### DISCUSSION TITLE
macos  minikube start error

### DISCUSSION THREAD

**Discussion by keweiLv** [guangzhengli/kubenates_workshop#12] [Q&A]
`[etcd] Creating static Pod manifest for local etcd in "/etc/kubernetes/manifests"
[wait-control-plane] Waiting for the kubelet to boot up the control plane as static Pods from directory "/etc/kubernetes/manifests". This can take up to 4m0s
[kubelet-check] Initial timeout of 40s passed.

Unfortunately, an error has occurred:
	timed out waiting for the condition

This error is likely caused by:
	- The kubelet is not running
	- The kubelet is unhealthy due to a misconfiguration of the node in some way (required cgroups disabled)

If you are on a systemd-powered system, you can try to troubleshoot the error with the following commands:
	- 'systemctl status kubelet'
	- 'journalctl -xeu kubelet'`

**Comment by guangzhengli** [comment]
What is your Mac version and chip? M1 or Intel?

And you can see more detail by `minikube logs`.

**Reply by keweiLv** [reply]
M1

**Reply by guangzhengli** [reply]
you can follow these commands first:
- 'systemctl status kubelet'
- 'journalctl -xeu kubelet'`

if it not working, you can show more details about errors by `minikube logs`

**Reply by keweiLv** [reply]
Although do not know the cause, the problem is solved. I restarted my computer and it started normally,thx.
